dotnet/agent-nr-config/Program.cs

```diff
@@ -0,0 +1,25 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseHttpsRedirection();
+
+app.UseAuthorization();
+
+app.MapControllers();
+
+app.Run();
```

Comment:
Its amazing that this file doesn't have any references to OpenTelemetry (never seen an instrumented C# app not have references here).  Might be cool to add a comment explicitly calling out there is no references to any opentelementry packages, to really drive the point home

Reply:
I'm thinking that it may make the most sense to call this out in the readme so that the application itself is entirely unmodified.